tests/scripts/run_tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/env bash
```

Comment:
why switch to bash ?

Reply:
no particular reason.

iirc, it \seemed to fix something.

Now changing it back to sh.